### DISCUSSION TITLE
I want to know how to modify the CSS parameter of el input group_ append in the input component

### DISCUSSION THREAD

**Discussion by unclezhuang** [element-plus/element-plus#16394] [Q&A] [answered]
When I was writing the page for Vue3, I used the input component and successfully modified the color of the input box through CSS overlay
![image](https://github.com/element-plus/element-plus/assets/119418630/46fba68c-4fce-4dac-b2a1-2f581e3d75fe)
This is my current code：
![image](https://github.com/element-plus/element-plus/assets/119418630/eac3871b-e39e-443c-b4cc-c6243491f74a)
As you can see, I can override the default CSS code by directly modifying the input in the CSS code section, but when I try to modify the. el input group_append, I cannot apply the changes
This is the environment on my webpage：
![image](https://github.com/element-plus/element-plus/assets/119418630/c84b2d17-6d42-4f7d-ae82-c3396b6023c0)
Modifying input will display in style
![image](https://github.com/element-plus/element-plus/assets/119418630/3f9fa72b-4472-4e50-b642-af7605e10398)
But el input group_append cannot be modified：
![image](https://github.com/element-plus/element-plus/assets/119418630/0d572c28-9038-4629-8226-6b2021995988)
my code：
   <el-input
               v-model="input"
                style="width: 240px"
                 placeholder="Enter price"
                 class="inputprice"
                 >
              <template #append>eth</template>
             </el-input>

**Comment by Aseisman** [comment, marked as answer]
![image](https://github.com/element-plus/element-plus/assets/38846995/8859c2bd-5155-4679-af71-bca45b65952b)

**Reply by Aseisman** [reply]
or
you can delete the "scoped" in the style. （it is basis....）and the next time you can show us the demo url
such as : 
https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbiAgPGVsLWlucHV0IHYtbW9kZWw9XCJpbnB1dFwiIHN0eWxlPVwid2lkdGg6IDI0MHB4XCIgcGxhY2Vob2xkZXI9XCJQbGVhc2UgaW5wdXRcIiBjbGFzcz1cImlucHV0cHJpY2VcIj5cbiAgICA8dGVtcGxhdGUgI2FwcGVuZD5ldGg8L3RlbXBsYXRlPlxuICA8L2VsLWlucHV0PlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcbmNvbnN0IGlucHV0ID0gcmVmKCcnKVxuPC9zY3JpcHQ+XG5cbjxzdHlsZSBzY29wZWQ+XG4uZWwtaW5wdXQge1xuICAtLWVsLWlucHV0LWJnLWNvbG9yOnJnYigyLDIsMSk7XG4gIC0tZWwtaW5wdXQtZm9jdXMtYm9yZGVyOiAjZjVmMGYwO1xuICAtLWVsLWlucHV0LWZvY3VzLWJvcmRlci1jb2xvcjogI2Y1ZjBmMDtcbn1cbi5pbnB1dHByaWNlIDpkZWVwKC5lbC1pbnB1dC1ncm91cF9fYXBwZW5kKSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICM0MTMxMzEgIWltcG9ydGFudDtcbiAgY29sb3I6ICNmZmZmZmYgIWltcG9ydGFudDtcbn1cbjwvc3R5bGU+IiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiX28iOnt9fQ==

**Reply by unclezhuang** [reply]
Thank you for your reply. It has solved my problem very well
